src/uct/api/uct.h

```diff
@@ -2037,14 +2038,14 @@ ucs_status_t uct_md_mem_dereg(uct_md_h md, uct_mem_h memh);
  * @param [in]     md           Memory domain to detect memory type
  * @param [in]     addr         Memory address to detect.
  * @param [in]     length       Size of memory
- * @param [out]    mem_type_p   Filled with memory type of the address range if
+ * @param [out]    mem_info_p   Filled with memory info of the address range if
                                 function succeeds
- * @return UCS_OK               If memory type is successfully detected
+ * @return UCS_OK               If memory info is successfully detected
  *         UCS_ERR_INVALID_ADDR If failed to detect memory type
  */
 ucs_status_t uct_md_detect_memory_type(uct_md_h md, const void *addr,
```

Comment:
maybe rename to `uct_md_detect_memory_info()` or don't change it, but add new API functions that returns memory info?

Reply:
I think we need to rename it to uct_md_detect_memory_info().